Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -11858,7 +11864,8 @@ def send_data_to_xsiam(data, vendor, product, data_format=None, url_key='url', n
     calling_context = demisto.callingContext.get('context', {})
     instance_name = calling_context.get('IntegrationInstance', '')
     collector_name = calling_context.get('IntegrationBrand', '')
-    items_count = len(data) if isinstance(data, list) else 1
+    if not items_count:
```

Comment:
let's add test case for both cases

Reply:
added test "test_send_data_to_xsiam_custom_snapshot_id_and_items_count"